src/middlewared/middlewared/plugins/kubernetes_linux/update.py

```diff
@@ -132,6 +134,14 @@ async def validate_data(self, data, schema, old_data):
                 'System is not licensed to use Applications'
             )
 
+        if data['passthrough_mode'] and (
+            license := self.middleware.call('system.license')
+        ) and '-MINI-' not in license['system_product']:
+            verrors.add(
+                f'{schema}.passthrough_mode',
+                'Can only be enabled on licensed iX enterprise hardware'
+            )
```

Comment:
Can't we just like: Checkbox this away, with a very clear warning popup.
A tad like what we will do with disabling hostPath validation?